--- tiny_mce_src.js.orig	2008-05-31 16:32:58.311195600 +0200
+++ tiny_mce_src.js	2008-05-31 16:33:22.903214200 +0200
@@ -1949,7 +1949,7 @@ TinyMCE_Engine.prototype = {
 					modal = "no";
 
 				if (template.close_previous != "no")
-					try {tinyMCE.lastWindow.close();} catch (ex) {}
+					try {if (typeof tinyMCE.lastWindow == 'object') tinyMCE.lastWindow.close();} catch (ex) {};
 
 				win = window.open(url, "mcePopup" + new Date().getTime(), "top=" + y + ",left=" + x + ",scrollbars=" + scrollbars + ",dialog=" + modal + ",minimizable=" + resizable + ",modal=" + modal + ",width=" + width + ",height=" + height + ",resizable=" + resizable);
 				if (win == null) {
